### ISSUE TITLE
Window width resize

### ISSUE THREAD

**Issue by kdedakia** [akiran/react-slick#67]
Resizing the window will not cause the slick carousel to change its width as expected. 

Resizing the window to be smaller will not shrink the carousel element.

Resizing the window to be bigger with a single item carousel will result in seeing portions of the next slide.

I'm not sure if this is the same as issue https://github.com/akiran/react-slick/issues/64

**Comment by iandoe** [comment]
Hello, you can fix this issue and the following issues: #64, #27 by creating a wrapper component for slick and giving it the following properties: 

```
  componentDidMount: function () {
    window.addEventListener('resize', this.onWindowResized);
  },

  componentWillUnmount:function () {
    window.removeEventListener('resize', this.onWindowResized);
  },

  onWindowResized: function () {
    // Optionnal debouncing using underscore
    _.debounce(
      this.forceUpdate()
    , 300);
  },
```

The component will rerender everytime the window is resized, and will then relayout.

**Comment by akiran** [comment]
I will fix this today

**Comment by akiran** [comment]
Fixed this in 0.6.3 release
